Taskfile.yml

```diff
@@ -33,7 +33,16 @@ tasks:
     desc: Build Yarn workspaces
     deps: ["workspaces:setup"]
     cmds:
-      - yarn wsrun --stages --report --fast-exit --exclude-missing ws:build
+      - |
```

Comment:
The requirement is to distribute the provided env to each workspace package that uses the env. Here's what is in the existing workflow:
```echo "OPTIC__NPM__VERSION=${{ needs.release_logic.outputs.version }}" >> "$FLAGS_FILE"
        if [[ -r $FLAGS_FILE ]]; then
          echo "Feature flags found for $RELEASE_CHANNEL, staging for build and runtime."
          cp $FLAGS_FILE .env
          cp $FLAGS_FILE workspaces/agent-cli/.env
          cp $FLAGS_FILE workspaces/ci-cli/.env
          cp $FLAGS_FILE workspaces/local-cli/.env
          cp $FLAGS_FILE workspaces/cli-server/.env
          cp $FLAGS_FILE workspaces/cli-scripts/.env
          cp $FLAGS_FILE workspaces/diff-engine/.env
          cp $FLAGS_FILE workspaces/ui/.env.production.local
        fi
```

I can make that change